woocommerce/woocommerce
# [Experimental] Rating Filter: clean code about dropdown mode

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR is a follow-up of https://github.com/woocommerce/woocommerce/pull/52152.
We decided to don't implement the `Dropdown` mode for this block, at least for this cycle.
This is a janitorial PR removing the code about the Dropdown representation of the block, and this, the multiple selection.

Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


0. Ensure the experimental blocks are enabled:
    - If you use the WooCommerce Beta Tester plugin, you can enable the experimental blocks in Tools > WCA Test Helper > Features. Toggle the `experimental-blocks` option.
    - If you build this PR locally, build the branch with `pnpm --filter='@woocommerce/plugin-woocommerce' build` command to have experimental blocks available.
1. Add the **Product Filters (Experimental)** block to the Product Catalog template.
2. Open the block sidebar
3. Confirm it doesn't show the Display section anymore   

Before | After
------|-----
<img width="284" alt="image" src="https://github.com/user-attachments/assets/090c53b7-d30f-4977-8394-de911c7b699a"> | <img width="281" alt="image" src="https://github.com/user-attachments/assets/f5522743-37d2-4749-b57c-0dcd14645f17">


5. Confirm the block keeps working as expected 
  -  Save and go to the shop page.
  - Select some filters, see filter blocks react, and update the filter options accordingly.
  - Interact with the **Rating** filter slider. See the filter updated accordingly.





### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
